install-dev/theme/views/content.php

```diff
@@ -0,0 +1,129 @@
+   * obtain it through the world-wide-web, please send an email
+   * to license@prestashop.com so we can send you a copy immediately.
+   *
+   * DISCLAIMER
+   *
+   * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+   * versions in the future. If you wish to customize PrestaShop for your
+   * needs please refer to https://devdocs.prestashop.com/ for more information.
+   *
+   * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+   * @copyright Since 2007 PrestaShop SA and Contributors
+   * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+   */
+?>
+
+<div id="contentInfosBlock">
+  <h2><?php echo $this->translator->trans('Content of your store', [], 'Install'); ?></h2>
+
+  <?php if (count($this->themes) === 1): ?>
+    <input type="hidden" value="<?php echo current($this->themes)->get('name'); ?>" name="theme" />
+  <?php else: ?>
+    <div class="field clearfix">
+      <label class="aligned"><?php echo $this->translator->trans('Installation of theme', [], 'Install'); ?></label>
+
+      <div class="contentinput">
+        <div class="themes-container">
+          <?php foreach ($this->themes as $theme): ?>
+            <div class="theme-card">
+              <label>
+                <input value="<?php echo $theme->get('name'); ?>" type="radio" name="theme" style="vertical-align: middle;" <?php if ($this->session->content_theme === $theme->get('name')): ?>checked="checked"<?php endif ?> autocomplete="off" />
+                <?php echo $theme->get('display_name'); ?>
+                <div class="image-block">
+                  <img src="../<?php echo $theme->get('preview'); ?>" alt="<?php echo $theme->get('display_name'); ?>">
+                </div>
+
+                <p class="theme-version">
+                  <strong><?php echo $this->translator->trans('Version:', [], 'Install'); ?></strong> <?php echo $theme->get('version'); ?>
+                </p>
+              </label>
+            </div>
+          <?php endforeach; ?>
+        </div>
+      </div>
+
+      <p class="userInfos aligned"><?php echo $this->translator->trans('Select the theme to install.', [], 'Install'); ?></p>
+    </div>
+  <?php endif; ?>
+
+  <div class="field clearfix">
+    <label class="aligned"><?php echo $this->translator->trans('Installation of demo products', [], 'Install'); ?></label>
+    <div class="contentinput">
+      <label>
+        <input value="1" type="radio" name="install-fixtures" style="vertical-align: middle;" <?php if ($this->session->content_install_fixtures): ?>checked="checked"<?php endif; ?> autocomplete="off" />
+        <?php echo $this->translator->trans('Yes', [], 'Install'); ?>
+      </label>
+      <label>
+        <input value="0" type="radio" name="install-fixtures" style="vertical-align: middle;" <?php if (!$this->session->content_install_fixtures): ?>checked="checked"<?php endif; ?> autocomplete="off" />
```

Comment:
Could you put the inline style in CSS ?

Reply:
It was already like that ^^' maybe in another PR